dll/cpl/sysdm/lang/bg-BG.rc

```diff
@@ -325,6 +325,10 @@ END
 
 STRINGTABLE
 BEGIN
+    IDS_EDITVARCAPTION "Edit "
```

Comment:
This is likely not going to work in all languages -- what if the "Edit" vs "New" needs to go somewhere else in the sentence?
The resource entries should probably just be the four full strings.

Reply:
Right.